.github/workflows/playground.yaml

```diff
@@ -0,0 +1,49 @@
+
+name: playground
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  playground:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
+      - uses: actions/checkout@v3
+
+      - name: Enable Corepack
+        run: corepack enable
+
+      - name: Setup Node.js 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: yarn
+
+      - name: Install dependencies and build
+        working-directory: ./examples/sdk-demo-example
+        env:
+          BASE_PATH: /accelbyte-web-sdk
```

Comment:
Considering the path change later, should we do `/accelbyte-typescript-sdk` here?

Reply:
(I'm going to merge this now first so we can eat the changesets and stuff)